core/chainlink-sgx.Dockerfile

```
@@ -44,17 +64,17 @@ COPY --from=builder /opt/intel/ /opt/intel/
 ARG ENVIRONMENT
 COPY --from=builder /go/bin/chainlink /usr/local/bin/
 COPY --from=builder \
-  /go/src/github.com/smartcontractkit/chainlink/sgx/target/$ENVIRONMENT/libadapters.so \
+  /chainlink/core/sgx/target/$ENVIRONMENT/libadapters.so \
   /usr/lib/
 COPY --from=builder \
-  /go/src/github.com/smartcontractkit/chainlink/sgx/target/$ENVIRONMENT/enclave.signed.so \
+  /chainlink/core/sgx/target/$ENVIRONMENT/enclave.signed.so \
   /root/
 
 # Launch chainlink via a small script that watches AESM + Chainlink
 ARG SGX_SIMULATION
 ENV SGX_SIMULATION $SGX_SIMULATION
 WORKDIR /root
-COPY ./chainlink-launcher-sgx.sh /root
+COPY core/chainlink-launcher-sgx.sh /root
```

I would either do this:
```suggestion
COPY core/chainlink-launcher-sgx.sh .
```
or remove line 76